tools/migrate/runner.go

```diff
@@ -220,6 +220,7 @@ func (r *Runner) Down(toRevertCount int) ([]string, error) {
 				return fmt.Errorf("Failed to save reverted migration info for %s: %w", m.file, err)
 			}
 
+			totalReverted++
```

Comment:
Let's remove this counter. It'll be better to rename the `applied` variable and change it declaration to `reverted := make([]string, 0, toRevertCount)` and use the `len(reverted)` instead.